scala/akkahttp/build.sbt

```diff
@@ -14,6 +14,7 @@ assemblyOption in assembly := (assemblyOption in assembly).value
 assemblyJarName in assembly := "server_scala_akkahttp"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.akka" %% "akka-http" % "[10.0,10.1)",
-  "com.typesafe.akka" %% "akka-http-core" % "[10.0,10.1)"
+  "com.typesafe.akka" %% "akka-http" % "10.1.11",
```

Comment:
could we use `[10.1,10.2)` in order to stick to **10.1** but have last up-to-date patch version ?